cider/exec/template/StringOpsIR.cpp

```diff
@@ -96,6 +96,55 @@ apply_string_ops_and_encode(const char* str_ptr,
   return string_dict_proxy->getOrAddTransient(result_str);
 }
 
+extern "C" RUNTIME_EXPORT int64_t
+apply_string_ops_and_encode_cider(const char* str_ptr,
+                                  const int32_t str_len,
+                                  const int64_t string_ops_handle,
+                                  const int64_t string_hasher_handle) {
```

Comment:
I think the `const` is not necessary for the three arguments.

Reply:
yes it's not necessary, but it could give user hints that they should not be modifiable.